### PR TITLE
Refactor string formatting to use f-strings

### PR DESCRIPTION
**Pull request by newwingbird**
Related to #657 

I will splite this modification into some PRs.
This issue's scope is only inside print and string literal.

### CODE REVIEW

File: comtypes/automation.py [hunk 1]
```diff
@@ -711,17 +711,7 @@ class tagEXCEPINFO(Structure):
...
-                self.wCode,
-                self.bstrSource,
-                self.bstrDescription,
-                self.bstrHelpFile,
-                self.dwHelpContext,
-                self.pfnDeferredFillIn,
-                self.scode,
-            ),
-        )
+        return f"<EXCEPINFO {self.wCode, self.bstrSource, self.bstrDescription, self.bstrHelpFile, self.dwHelpContext, self.pfnDeferredFillIn, self.scode}>"
```
Comment (junkmd) [inline comtypes/automation.py:714]: If the line becomes too long, I believe we could make use of a temporary variable.
```suggestion
        info = (self.wCode, self.bstrSource, self.bstrDescription, self.bstrHelpFile, self.dwHelpContext, self.pfnDeferredFillIn, self.scode)
        return f"<EXCEPINFO {info}>"
```
The contents of the tuple will actually be formatted by `ruff`, with appropriate line breaks inserted as needed.

File: comtypes/_memberspec.py [hunk 2]
```diff
@@ -175,14 +175,15 @@ def COMMETHOD(idlflags, restype, methodname, *argspec) -> _ComMemberSpec:
...
-        name = methodname
+    name = (
+        f"_get_{methodname}"
+        if "propget" in idlflags
+        else f"_set_{methodname}"
+        if "propput" in idlflags
+        else f"_setref_{methodname}"
+        if "propputref" in idlflags
+        else methodname
+    )
```
Comment (junkmd) [inline comtypes/_memberspec.py:186]: If [conditional expressions (aka "ternary operators")](https://docs.python.org/3/reference/expressions.html#conditional-expressions) becomes long enough that it is broken into multiple lines by the formatter, it is likely more readable to use regular [`if` statements](https://docs.python.org/3/tutorial/controlflow.html#if-statements) instead.

File: comtypes/_post_coinit/_cominterface_meta_patcher.py [hunk 3]
```diff
@@ -17,7 +17,7 @@ def __getattr__(self, name):
             try:
                 fixed_name = self.__map_case__[name.lower()]
             except KeyError:
-                raise AttributeError(name)  # Should we use exception-chaining?
+                raise AttributeError(f"{name}")  # Should we use exception-chaining?
```
Comment (junkmd) [inline comtypes/_post_coinit/_cominterface_meta_patcher.py:20]: Is this necessary?

**Review by junkmd — CHANGES_REQUESTED** [review]

File: comtypes/automation.py (hunk 1 above)
Comment (newwingbird) [reply comtypes/automation.py]: Good suggestion! 
Using "suggested change" is my first time.

**Review by junkmd — APPROVED** [review]
Thanks.

**Pull request merged**